modules.d/90overlay-root/overlay-mount.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# make a read-only nfsroot writeable by using overlayfs
+# the nfsroot is already mounted to $NEWROOT
+# add the parameter rootovl to the kernel, to activate this feature
+
+. /lib/dracut-lib.sh
+
+if ! getargbool 0 rootovl ; then
```

Comment:
This should probably become `rd.rootovl` (and keep `rootovl` as a compat parameter). @haraldh?